### DISCUSSION TITLE
subscriptions add-on

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#951] [General]
Do you know if there is a subscriptions manager add-on?

**Comment by karandatwani92** [comment]
Hey @blondie63 

Backpack doesn't currently have a subscription manager add-on. If you have features/ideas, you can suggest them [here](https://github.com/Laravel-Backpack/community-forum/discussions/751). It might get launched someday.

**Comment by blondie63** [comment]
What you think about https://laravel.com/docs/11.x/billing ?

**Reply by karandatwani92** [reply]
How do you imagine it?
